Feature/281/events page mouse over

JavaOfDoom opened the pull request:
### Issue: #281 

### Describe the problem being solved: 
Description of event displayed in mouse-over overlay was shortened to 200 characters to fit on approximately 5 lines and trimmed at the last blank space to avoid words being cut in half. An ellipsis was also added to show that there is more to the description.
Text was added to the bottom of the overlay in the style of a button and made to match the rest of the aesthetic of the site. The text and border turn yellow on hover over the button like the links on the rest of the site.
Additionally opacity of the overlay was increased to promote a more easily readable design 
![Screenshot from 2019-11-14 11-58-31](https://user-images.githubusercontent.com/36250779/68884100-0ed89900-06d8-11ea-8120-09e0797ae8de.png)

### Impacted areas in the application: 
Events page event cards mouse-over overlay

List general components of the application that this PR will affect: 
* 

PR checklist
- [x] I included  a screenshot for FE changes
- [x] I have linked the PR to a Zenhub ticket
- [x] I have checked for merge conflicts
- [x] I have run the [prettier](https://prettier.io/) command `make pretty`

dkaplan88 commented on frontend/src/components/events/molecules/event_card.css:
```diff
@@ -18,14 +18,39 @@
+  position: absolute;
+  width: max-content;
+  bottom: 20px;
+  right: 97.5px;
```
we can do this a bit easier by creating a parent div

so `<div class="learnMore"><btn></btn></div>`

then do this css like so 

```
.learnMore {
    position: absolute;
    bottom: 20px;
    width: 100%;

    btn {
          position: relative;
          margin: auto;
         display: flex;
         background: #519b56;
         border-color: #eeeaea;
         color: #eeeaea;
         font-weight: bold;
    }
}
```

dkaplan88 commented on frontend/src/components/events/organisms/event_section.jsx:
```diff
@@ -17,6 +17,18 @@ const EventSection = ({ events, isUpcoming }) => {
         </div>
         {events.map(({ name, start, logo, url, id, description }) => {
+          const maxChar = 200;
+          if (description.text.length > maxChar) {
```
this can be written a bit more succinctly

```
description.text = description.text.length > 200 ? description.text.substring(0, 200) + '...' : description.text
```

JavaOfDoom commented:
I put the button in a parent div and adjusted the styling. I also updated the if statement so that is more succinct. I also had to then change the portion below it so that it didn't add the ellipsis to ever description and only to the long ones. I updated npm, ran make pretty, and merged staging in again.

JavaOfDoom commented:
Abstraction of the method to shorten the description on the events mouse over is completed. There were some merge conflicts with package-lock.json that I believe I resolved properly. I will update again on my end. Ready for another review.

JavaOfDoom commented:
Changed the method to use split, slice, and join run more efficiently.
